packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -151,6 +151,16 @@ const HeaderTools = ({
                     >
                       PatternFly 3
                     </DropdownItem>
+                    <DropdownItem
+                      key="PatternFly 4"
+                      className="ws-patternfly-3"
+                      isExternalLink
+                      to="http://v4-archive.patternfly.org/v4/"
+                      itemId="patternfly-4"
+                    >
+                      PatternFly 4
+                      <ExternalLinkAltIcon />
```

Comment:
Looks like this icon is not longer imported & no longer needed with the `isExternalLink` prop that's used.

```suggestion

```